### PR TITLE
fix version field for core.v0.12.2

### PR DESCRIPTION
**Pull request by damiendoligez**
Fix a copy-paste mistake.

cc @janestreet

### CODE REVIEW

**Comment by mseri** [comment]
Ideally we should really remove that field

File: packages/core/core.v0.12.2/opam [hunk 1]
```diff
@@ -1,5 +1,5 @@
 opam-version: "2.0"
-version: "v0.12.1"
+version: "v0.12.2"
```
Comment (mseri) [inline packages/core/core.v0.12.2/opam]:
```suggestion
```

Comment (kit-ty-kate) [reply packages/core/core.v0.12.2/opam]: (sorry I wasn't able to commit your suggestion for some unknown reason) but thanks!!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**